gobblin-data-management/src/main/java/org/apache/gobblin/data/management/version/finder/AbstractDatasetVersionFinder.java

```diff
@@ -17,21 +17,22 @@
 
 package org.apache.gobblin.data.management.version.finder;
 
+import com.google.common.collect.Lists;
```

Comment:
gobblin using a different codestyle and import orders, please use this https://gobblin.readthedocs.io/en/latest/developer-guide/CodingStyle/

Reply:
you need to import codestyle-intellij-gobblin.xml file in your IDE

Reply:
Done, thanks for pointing out